### PR TITLE
Update gd article

### PR DESCRIPTION
**Pull request by shailenpatel2**

### CODE REVIEW

File: cc-by-sa/knowledge/contributors.textproto [hunk 1]
```diff
@@ -30,6 +30,10 @@ contributors {
   name: "Cassidy Curtis"
   personal_site: "http://otherthings.com"
 }
+contributors {
+  name: "Charles Berret"
+  personal_site: ""
```
Comment (davelab6) [inline cc-by-sa/knowledge/contributors.textproto:35]: https://charlesberret.net/

File: cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: "Choosing web fonts: A beginner's all-in-one guide"
-authors: "Dave Crossland"
+authors: "Charles Berret"
 reviewers: "Dave Crossland"
```
Comment (davelab6) [inline cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto:3]: Yuin Chien, Joana Correia, Dave Crossland, Natanael Gama, Octavio Pardo, Eben Sorkin, and Eduardo Tunni

File: cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/content.md [hunk 3]
```diff
@@ -113,4 +113,4 @@ Typography is a subtle art, but less elusive than its reputation might suggest.
 
 Several designers lent their expertise to the construction of this guide: Yuin Chien, Joana Correia, Dave Crossland, Natanael Gama, Octavio Pardo, Eben Sorkin, and Eduardo Tunni.
```
Comment (davelab6) [inline cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/content.md:114]: Can delete this as its handled by the template with the `reviewers` key

File: cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto (hunk 2 above)
Comment (shailenpatel2) [reply cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto:4]: @davelab6 do we know about personal sites for these reviewers? I can add them in as well

Comment (shailenpatel2) [reply cc-by-sa/knowledge/modules/choosing_type/lessons/choosing_web_fonts_beginners_guide/lesson.textproto:4]: Reviewer personal sites can be added in a followup after confirming them with Dave async

**Review by nathan-williams — APPROVED** [review]

**Pull request merged**